client/verta/verta/tracking/entities/_entity.py

```diff
@@ -216,7 +216,7 @@ def _create_proto_internal(
         tags=None,
         attrs=None,
         date_created=None,
-        **kwargs
+        **kwargs,
```

Comment:
Throwback to when this trailing comma was [a syntax error in Python ≤3.5](https://stackoverflow.com/questions/33350454/syntaxerror-with-passing-kwargs-and-trailing-comma).